script.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+let total = 0;
+let highest = 0;
+let stuAvg= 0;  
+var names= [];
+var grade= [];
+var pos1 = [];
+var arr = [];
+
+for (var i = 0; i < 10; i++) {             
+    names.push(prompt('Enter The Ten Student Names | Student: ' + (i + 1))); 
+    document.getElementById("stuNames").innerHTML=("Student's Names:" +names.join(','));
+          
+    grade.push(Number(prompt('Enter The Ten Student Grades | Grade: ' + (i + 1))));
+    total += grade[i] 

```

Comment:
You could save the prompt answer to a variable like ```var grade = Number(prompt('Enter The Ten Student Grades | Grade: ' + (i + 1)))``` and then push that to the grades array and add the variable to the total. Could cut down on future debugging time.